src/tests/JIT/Methodical/Arrays/range/float64_range1.il

```diff
@@ -20,19 +20,13 @@
            
   
 }
-.assembly 'test'
+.assembly 'float64_range1'
 {
-
-
-  
-  
-  
 }
 .assembly extern xunit.core {}
-.module 'test.exe'
```

Comment:
Is this a remaining cleanup we should apply to all ilproj tests (putting .assembly in sync with the source file, removing the .module line)? I believe I should be able to tweak my IL rewriter to bulk process all runtime ilproj tests like this.

Reply:
Yes, this cleanup is actually required for ProjectReferences to il projects to work as expected. Otherwise, the SDK's conflict resolution will throw away all but one referenced assembly.